lib/compute/operation.js

```diff
@@ -268,10 +275,16 @@ Operation.prototype.startPolling_ = function() {
     }
 
     if (metadata.status !== 'DONE') {
+      if (metadata.status === 'RUNNING' && !self.status) {
```

Comment:
Could you move this above the "if !== DONE" block? It's a little easier to follow along when it's not nested.